File: C++/Database/conn.cpp
#include<iostream.h>
int main()
{
sql::mysql::MySQL_Driver *driver;
sql::Connection	*con;

driver = sql::mysql::MySQL_Driver::get_mysql_driver_instance();
con = driver->connect("http://127.0.0.1:3306", "root", "");
cout<<"Connected";
delete con;
return 0;
}

File: perl/Database/conn.pl
# PERL MODULE
use Mysql;

# HTTP HEADER
print "Content-type: text/html \n\n";

# CONFIG VARIABLES
$host = "localhost";
$database = "amit";
$tablename = "emp";
$user = "root";
$pw = "";

# PERL MYSQL CONNECT
$connect = Mysql->connect($host, $database, $user, $pw);

print "Connected Successfully\n";

@databases = $connect->listdbs;
foreach $database (@databases) {
	print "$database\n";
}

@tables = $connect->listtables;
foreach $table (@tables) {
	print "$table\n";
}